packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -54,9 +93,12 @@ export const NativeSelect = React.forwardRef<HTMLDivElement, NativeSelectProps>(
       ...other
     } = props;
     const theme = React.useContext(ThemeContext);
+    const i18n = React.useContext(I18nContext);
     const isInverse = useIsInverse(isInverseProp);
     const id = useGenerateId(defaultId);
 
+    const nativeSelectLabel = i18n.select.placeholder;
```

Comment:
Accessibility wise, should the aria-label be the placeholder (in this case `Select...`) or the `labelText`?

Reply:
Removed from component as deemed unnecessary by Leslie.